### DISCUSSION TITLE
Bot.health is not a function?

### DISCUSSION THREAD

**Discussion by AshDoesStuf** [PrismarineJS/mineflayer#2643] [Q&A]
TypeError: bot.health is not a function

when i run the bot it gives me an error
heres code:
function checkHealth() {
  if(bot.health() <= 10){
      station()
      return bot.chat('...')
  } else if (bot.health() === 20){
    return console.info('bot is ok')
  }
}

**Comment by amoraschi** [comment]
Error tells you, it's not a function, it's a property

```js
bot.health
```

Hate to be that guy, but, read the docs

**Reply by AshDoesStuf** [reply]
bruh
nvm ill do everything my self
